packages/peregrine/lib/talons/CheckoutPage/useCheckoutPage.js

```diff
@@ -118,6 +124,7 @@ export const useCheckoutPage = props => {
 
         // Delete stale cart data from apollo
         await deleteCacheEntry(apolloClient, key => key.match(/^Cart/));
+        await deleteCacheEntry(apolloClient, key => key.match(/^\$Cart/));
```

Comment:
Merge the rules together, suggestions above.

Reply:
acf791049